Blockzilla/SettingsViewController.swift

```diff
@@ -109,39 +109,50 @@ class SettingsTableViewToggleCell: SettingsTableViewCell {
     private let newDetailLabel = SmartLabel()
     private let spacerView = UIView()
     var navigationController: UINavigationController?
-    
+
     init(style: UITableViewCell.CellStyle, reuseIdentifier: String?, toggle: BlockerToggle) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
-        
+
         newLabel.numberOfLines = 0
         newLabel.text = toggle.label
+
+        setupDynamicFont()
+
+        if #available(iOS 10.0, *) {
```

Comment:
This is likewise duplicate and could be a nice function (or even perhaps just included in setupDynamicFont).